<launch>
  <arg name="robot" default="scitos_coverage"/>

  <arg name="exp" default="ecai"/>
  <arg name="env" default="1"/> <!-- 1, 2, 3, 4 -->
  <arg name="soma_roi" default="1"/> <!-- 1, 3, 4, 5 -->

  <arg name="input_file" default="$(find viper)/data/$(arg exp)/env$(arg env)/views.json"/>

  <arg name="output_file_values" default="$(find viper)/data/$(arg exp)/env$(arg env)/view_values.json"/>
  <arg name="output_file_costs" default="$(find viper)/data/$(arg exp)/env$(arg env)/view_costs.json"/>
  <arg name="output_file_view_keys" default="$(find viper)/data/$(arg exp)/env$(arg env)/view_keys.json"/>


  <node name="octomap" pkg="octomap_server" type="octomap_server_node" args="$(find strands_morse)/aloof/maps/ecai$(arg env).bt"/>

  <include file="$(find viper)/launch/$(arg robot)/_evalaute_views.launch"/>

  <node name="evaluate_views" pkg="viper" type="eval_views.py" output="screen">
    <param name="robot" type="string" value="$(arg robot)"/>
    <param name="input_file" type="string" value="$(arg input_file)"/>
    <param name="output_file_values" type="string" value="$(arg output_file_values)"/>
    <param name="output_file_costs"  type="string" value="$(arg output_file_costs)"/>
    <param name="output_file_view_keys" type="string" value="$(arg output_file_view_keys)"/>
  </node>
</launch>
